Wikipedia/UI-V5/WMFTableOfContentsViewController.swift

```diff
@@ -111,6 +113,7 @@ public class WMFTableOfContentsViewController: UITableViewController, WMFTableOf
     public override func viewDidLoad() {
         super.viewDidLoad()
         let header = WMFTableOfContentsHeader.wmf_viewFromClassNib()
+        header.articleSite = delegate?.tableOfContentsArticleSite()
```

Comment:
we may want to assert that the delegate is set in `viewDidLoad` now since we require it for the header.